test/vagrant-ci-start.sh

```diff
@@ -4,6 +4,10 @@ set -e
 
 K8S_NODES=${K8S_NODES:-2}
 
+echo "restarting portmap and nfs-kernel-server services to combat nfs server issues"
+systemctl restart portmap.service || true
+systemctl restart nfs-kernel-server.service || true
```

Comment:
Why `|| true`? Don't we want to fail if restarting the service fails?

Reply:
Restarting `nfs-kernel-server` can actually fail while nfs server is running with this error message:
```
Job for nfs-server.service canceled.
```
, which means that everything is fine and we should continue.